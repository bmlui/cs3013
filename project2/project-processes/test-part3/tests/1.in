/bin/ls
history
echo hey there
cd /
pwd
history
exit

