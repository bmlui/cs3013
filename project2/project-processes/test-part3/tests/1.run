PATH="/bad" ./wshell < tests/1.in
