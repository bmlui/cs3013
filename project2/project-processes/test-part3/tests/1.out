test-part3$ /bin/ls
test_part3.sh
tests
tests-out
wshell
test-part3$ history
/bin/ls
history
test-part3$ echo hey there
hey there
test-part3$ cd /
/$ pwd
/
/$ history
/bin/ls
history
echo hey there
cd /
pwd
history
/$ exit
